pkg/rpc/server_test.go

```diff
@@ -112,6 +112,38 @@ var rpcTestCases = map[string][]rpcTestCase{
 			fail:   true,
 		},
 	},
+	"gettxout": {
```

Comment:
Hmm, and where is the case with some successful txout result?

Reply:
It is tested separately below.